packages/cozy-pouch-link/src/CozyPouchLink.js

```diff
@@ -556,6 +556,15 @@ class PouchLink extends CozyLink {
       res = withoutDesignDocuments(res)
       withRows = true
     } else {
+      if (indexedFields) {
+        for (const indexedField of indexedFields) {
+          if (!Object.keys(selector).includes(indexedField)) {
+            selector[indexedField] = {
```

Comment:
I feel like we should add a warning here, that we do some magic to please pouchdb but that ideally the query should be fixed

Reply:
And we probably should extract this logic into a function with a test :wink:

Reply:
Done in #1524